contact.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kontakt</title>
+</head>
+
+<body>
+    <header>
+        <h1>Kontakt</h1>
+        <nav>
+            <a href="index.html">Start</a>
+            <a href="linz.html">Linz</a>
+            <a href="oslo.html">Oslo</a>
+            <a href="london.html">London</a>
+            <a href="stPetersburg.html">St. Petersburg</a>
+            <a href="warschau.html">Warschau</a>
+            <a href="uppsala.html">Uppsala</a>
+            <a href="montreal.html">Montreal</a>
+            <a class="current" href="contact.html">Kontakt</a>
+        </nav>
+    </header>
+
+    <main>
+        <form action="">
+            <label for="name">Name:</label>
+            <input type="text" name="name" placeholder="Ben Gombl">

```

Comment:
wenn du beim input die id mit dem Wert vom label aus for setzt, verbindest du die beiden. Heißt: wenn du auf den label klickst wird der input aktiv. ist für mobil sehr sehr nützlich